examples/pl_basics/autoencoder.py

```diff
@@ -50,6 +50,7 @@ def __init__(
     ) -> None:
         """
         Args:
+            HERE
```

Comment:
```suggestion
```

Reply:
Yes, this is there to demo it resolves the bug. I will remove it before merging ;)

Reply:
Let's wait for it to go through.